pkg/infra/baseagent.go

```diff
@@ -49,6 +49,14 @@ func (a *Base) HP() int {
 	return a.hp
 }
 
+func (a *Base) CurrPlatFood() float64 {
+	foodOnPlatform, platformFloor := a.tower.GetCurrPlatFood()
```

Comment:
```suggestion
	foodOnPlatform, platformFloor := a.tower.CurrPlatFood()
```

Reply:
I've bypassed putting functions in tower as Jason suggested below so this is no longer an issue.